packages/vow/src/types.js

```diff
@@ -122,3 +122,11 @@ export {};
  * @template [T=any]
  * @typedef {import('./types.js').ERef<T | import('./types.js').Vow<T>>} Specimen
  */
+
+/**
+ * @typedef {object} VowTools
+ * // TODO type according to prepareWatch returns
```

Comment:
Do you mean something like:
```suggestion
 * @property {import('./watch.js).Watch} watch
 * @property {import('./when.js').When} when
 * @property {<T = any>() => VowKit<T>} makeVowKit
```

Reply:
I did indeed, thanks!

Done.